Makefile

```diff
@@ -7,10 +7,17 @@ BINDIR ?= $(PREFIX)/bin
 VERSION="0.3.1"
 
 CFLAGS ?= -Wall -Wextra -Wno-unused-parameter -Wno-parentheses
+CFLAGS += `pkg-config --cflags xkbcommon` \
+	`pkg-config --cflags wayland-client` \
+	`pkg-config --cflags wayland-cursor`
 override CFLAGS += -DVERSION=\"$(VERSION)\"
 
 VPATH=$(WAYLAND_PROTOCOLS_DIR)/stable/xdg-shell
 LIBS=-lrt -lm -lutil -lwayland-client -lwayland-cursor -lxkbcommon -Ltsm -lhtsm
```

Comment:
Some of these can be removed then if they are added by pkg-config instead.

Reply:
True, I'll clean that up.